sdk/fabric/azure-resourcemanager-fabric/src/main/java/com/azure/resourcemanager/fabric/fluent/FabricCapacitiesClient.java

```diff
@@ -479,4 +480,29 @@ PagedIterable<RpSkuDetailsForExistingResourceInner> listSkusForCapacity(String r
      */
     @ServiceMethod(returns = ReturnType.COLLECTION)
     PagedIterable<RpSkuDetailsForNewResourceInner> listSkus(Context context);
+
+    /**
+     * List the current consumption and limit in this location for the provided subscription.
+     * 
+     * @param location The location name.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws com.azure.core.management.exception.ManagementException thrown if the request is rejected by server.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return paged collection of Quota items as paginated response with {@link PagedIterable}.
+     */
+    @ServiceMethod(returns = ReturnType.COLLECTION)
+    PagedIterable<QuotaInner> listUsages(String location);
```

Comment:
Same here - we should validate if the service is ready to release these new APIs.